seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1217,7 +1217,7 @@
         "We can also use the <code>+=</code> operator to <dfn>concatenate</dfn> a string onto the end of an existing string variable. This can be very helpful to break a long string over several lines.",
         "<strong>Note</strong><br>Watch out for spaces. Concatenation does not add spaces between concatenated strings, so you'll need to add them yourself.",
         "<hr>",
-        "Build <code>myStr</code> over several lines by concatenating these two strings:<br><code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator."
+        "Build <code>myStr</code> over several lines by concatenating these two strings: <code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator. Use the <code>+=</code> operator similar to how it is shown in the editor. Start by assigning the first string to <code>myStr</code>, then add on the second string."
```

Comment:
To clarify, my main concern was with the word order in the "then add on the second string", which I would write as "then add the second string on to the end of the first string" (but after writing that out, it doesn't seem quite right either 😕). If the word order is correct as "then add on", then this is good to merge to me.